.travis.yml

```diff
@@ -2,7 +2,8 @@ language: scala
 jdk:
   - oraclejdk8
```

Comment:
Just concerned we may not catch a java 7 jar sneaking in, tho I guess your Java Driver build will break :)